docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,135 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+InfluxDB
+========
+
+The Sawtooth team provides Docker files for Grafana and InfluxDB for reporting statistics and monitoring network health.
```

Comment:
If you add a purpose statement at the beginning, I'd move this sentence right after it.